README.md

```
@@ -83,18 +84,20 @@ cd build
+> _ClusterComplete_ can be made significantly faster by passing `-DFICTION_ENABLE_JEMALLOC=ON`, i.e.:
 
-> *ClusterComplete* can be made significantly faster by passing `-DFICTION_ENABLE_JEMALLOC=ON`, i.e.:
 ```bash
 cmake . -B build -DFICTION_ENABLE_JEMALLOC=ON
 cd build
 cmake --build . -j4
 ```
-> **NB:** This CMake option should not be passed when building *QuickSim* as it has an adverse effect on the runtime for this plugin.
+
+> **NB:** This CMake option should not be passed when building _QuickSim_ as it has an adverse effect on the runtime for this plugin.
 > Find more information [here](https://fiction.readthedocs.io/en/latest/getting_started.html#usage-of-jemalloc).
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)